### DISCUSSION TITLE
Welcome to height-app-api Discussions!

### DISCUSSION THREAD

**Discussion by beomjungil** [beomjungil/height-app-api#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by redraskal** [comment]
Thank you for writing this. I was in the middle of doing that and found your project :) I'm working on a Discord webhook because Height's integration, for some reason, lacks task update notifications.

Something seems to be off- VS Code cannot resolve the types. WebStorm can resolve the types but still displays an error.

```
TS7016: Could not find a declaration file for module 'height-app-api'.
'xxxxxxxx/node_modules/height-app-api/dist/es/index.js' implicitly has an 'any' type.   Try `npm i --save-dev @types/height-app-api` if it exists or add a new declaration (.d.ts) file containing `declare module 'height-app-api';`
```
